src/assessment.js

```diff
@@ -14,20 +14,20 @@
 /*8*/ };
 
 // Suppose we had declared another function between line 7 and 8. Would that
-// new function have access to the value stored in the variable `laseQuantity`
+// new function have access to the value stored in the variable `laserQuantity` **typo**
 // on line 3?
 //
 //     `true` or `false`
 //
-var q1 = /* your answer here */;
+var q1 = 'true'; //within scope of deathStar variable
```

Comment:
Correct answer, but wrong reason. Objects don't have scope, only functions do; as such `deathStar` and all of its properties are in the `global` scope.